proto/user-service.proto

```diff
@@ -52,6 +64,20 @@ message UpdateUserResponse {
     User user = 1;
 }
 
+message ValidateTokenResponse {
+    bool valid = 1;
+}
+
+message LoginRequest {
+    string username = 1;
```

Comment:
Logout requires a token to verify that the actual user is trying to logout, can remove that if you think it's unneccecary